src/Symfony/Component/Ldap/Adapter/EntryManagerInterface.php

```diff
@@ -40,4 +40,13 @@ public function update(Entry $entry);
      * @param Entry $entry
      */
     public function remove(Entry $entry);
+
+    /**
+     * Renames an entry on the Ldap server.
+     *
+     * @param Entry  $entry
+     * @param string $newRdn
+     * @param bool   $deleteOldRdn
+     */
+    public function rename(Entry $entry, $newRdn, $deleteOldRdn = true);
```

Comment:
Rename the `$deleteOldRdn` to `$removeOldRdn` so it is the same as in the implementation class.